README.md

```diff
@@ -334,6 +334,16 @@ When run with this flag, `publish` will publish to npm without running any of th
 
 > Only publish to npm; skip committing, tagging, and pushing git changes (this only affects publish).
 
+#### --skip-git-push
+
+```sh
+$ lerna publish --skip-git-push
+```
+
+When run with this flag, `publish` will publish to npm without pushing the commit and tags up to git remote.
```

Comment:
I must admit my gut-reaction to this PR was "this is going to be abused and it will cause far more damage than it is worth." Let me explain.

Lerna's core value proposition, from my perspective, is that it handles the boring, repetitive, but _necessary_ steps of managing multiple npm packages according to community best-practices. One of those best-practices is that (if you're managing the version field in source control) you should always push the version bump commit and git tag to origin after calling `npm publish`. `npm` does not _force_ you to do this, but it is nonetheless important to keep public git history consistent with public npm registry state.

Until we get to a place where lerna's internals are properly modularized and published as separate packages, I'm very reticent to add a footgun like this to address a perplexing CI requirement. If you've already accepted that a CI process will be injecting commits into your public history, what can't you accomplish with the current all-in-one approach? Adding generated changelogs to the commit?

Reply:
@evocateur an issue i've come across with Lerna pushing a commit along with a tag when using CI, is it results in perpetual running of builds. CI runs lerna publish, lerna pushes to git, CI detects a git change, and triggers a new build, and the cycle continues.

Am i missing something? I'm finding it very difficult with the current flags to get Lerna working nicely with CI. You must have an ideal workflow in your head when integrating Lerna into CI, what is it?